app/controllers/idv/doc_auth_controller.rb

```diff
@@ -17,13 +17,31 @@ class DocAuthController < ApplicationController
     }.freeze
 
     def doc_capture_poll
+      result = if FeatureManagement.document_capture_step_enabled?
+                 document_capture_session_poll_render_result
+               else
+                 doc_capture_poll_render_result
+               end
+
+      render result
+    end
+
+    def doc_capture_poll_render_result
       doc_capture = DocCapture.find_by(user_id: user_id)
-      render plain: 'Not authorized', status: :not_authorized if doc_capture.blank?
-      if doc_capture.acuant_token.present?
-        render plain: 'Complete', status: :ok
-      else
-        render plain: 'Pending', status: :accepted
-      end
+      return { plain: 'Not authorized', status: :not_authorized } if doc_capture.blank?
+      return { plain: 'Pending', status: :accepted } if doc_capture.acuant_token.blank?
+      { plain: 'Complete', status: :ok }
+    end
+
+    def document_capture_session_poll_render_result
+      session_uuid = flow_session[:document_capture_session_uuid]
+      document_capture_session = DocumentCaptureSession.find_by(uuid: session_uuid)
+      return { plain: 'Not authorized', status: :not_authorized } unless document_capture_session
+
+      result = document_capture_session.load_result
+      return { plain: 'Pending', status: :accepted } if result.blank?
+      return { plain: 'Not authorized', status: :not_authorized } unless result.success?
+      { plain: 'Complete', status: :ok }
```

Comment:
I'm seeing this didn't change here, but I'd love to follow up and make a separate API controller/endpoint that returns like a JSON object for this? Not urgent

Reply:
> I'm seeing this didn't change here, but I'd love to follow up and make a separate API controller/endpoint that returns like a JSON object for this? Not urgent

That makes sense to me! I created a tracking ticket at LG-3423.

At some point in the future, if the `document_capture_step_enabled` feature flag is treated as the default and removed, the logic here should be able to be collapsed back to about what it was previously. Even still, I'd think it makes sense as its own thing.

Reply:
>That makes sense to me! I created a tracking ticket at LG-3423.

For posterity's sake, the follow-up: #4293